src/components/ItaliaTheme/View/Commons/SideMenu.jsx

```diff
@@ -142,83 +152,54 @@ const SideMenu = ({ data, content_uid }) => {
-          aria-expanded={isNavOpen ? 'true' : 'false'}
-          aria-label={intl.formatMessage(messages.buttonToggle)}
-          data-target="#navbarNavB"
-          onClick={() => onNavScrollToggle()}
-        >
-          <span className="it-list"></span>
-          {intl.formatMessage(messages.index)}
-        </button>
-
-        <div
-          className={
-            isNavOpen ? 'navbar-collapsable expanded' : 'navbar-collapsable'
-          }
-          id="navbarNavB"
-          style={isNavOpen ? { display: 'block' } : { display: 'none' }}
-        >
-          <div
-            className="overlay"
-            style={isNavOpen ? { display: 'block' } : { display: 'none' }}
-          ></div>
-          <div className="close-div sr-only">
-            <button className="btn close-menu" type="button">
-              <span className="it-close"></span>
-              {intl.formatMessage(messages.close)}
-            </button>
-          </div>
-          <a
-            className="it-back-button"
-            href="#"
-            style={isNavOpen ? { display: 'block' } : { display: 'none' }}
-            onClick={(e) => {
-              e.preventDefault();
-              onNavScrollToggle();
-            }}
-          >
-            <Icon
-              className="align-top"
-              color="primary"
-              icon="it-chevron-left"
-              style={{ ariaHidden: true }}
-              size="sm"
-            />
-            <span>{intl.formatMessage(messages.back)}</span>
-          </a>
-          <div className="menu-wrapper">
-            <div className="link-list-wrapper menu-link-list">
-              <h3>{intl.formatMessage(messages.index)}</h3>
-              <ul className="link-list" data-element="page-index">
-                {headers.map((item, i) => (
-                  <li className="nav-item" key={item.id}>
-                    <a
-                      className={`nav-link ${
-                        item.id === activeSection && 'active'
-                      }`}
-                      href={`#${item.id}`}
-                      onClick={handleClickAnchor(item.id)}
-                      id={`item-${item.id}`}
-                    >
-                      {item.title}
-                    </a>
-                  </li>
-                ))}
-              </ul>
+        <div className="menu-wrapper">
+          <div className="link-list-wrapper menu-link-list">
+            <div className="accordion-wrapper">
+              <Accordion>
+                <AccordionHeader
+                  active={isNavOpen}
+                  onToggle={() => {
+                    setIsNavOpen(!isNavOpen);
+                  }}
+                  aria-controls="navbarNavB"
+                  aria-expanded={isNavOpen ? 'true' : 'false'}
+                  aria-label={intl.formatMessage(messages.buttonToggle)}
+                  data-target="#navbarNavB"
+                  className="accordion-button"
```

Comment:
Tutti questi non ci sono sulla v3, ma so anche che i componenti Accordion del kit sono diversi tra le due versioni, quindi ti chiedo conferma che servano, giusto per scrupolo. In caso mandami un link a dove li hai trovati come esempio così vedo anche io la differenza, grazie